docs/configuration.md

```diff
@@ -8,25 +8,15 @@ The full list of available configuration options can be seen in [config-defaults
 
 ## Authentication options
 
-Currently supported authentication methods are SAML2 and LDAP. Login method to be used can be defined with the key `:authentication` and the following values are recognized:
+Currently supported authentication methods are SAML2 and OpenId Connect (e.g. Auth0). Login method to be used can be defined with the key `:authentication` and the following values are recognized:
 
 * `:shibboleth` for SAML2
-* `:ldap`
 * `:fake-shibboleth` for development login
```

Comment:
:oidc is missing from this list

Reply:
I noticed the same and was planning to follow up with an OIDC documentation PR